src/Compilers/CSharp/Test/Semantic/Semantics/UninitializedNonNullableFieldTests.cs

```diff
@@ -2361,6 +2361,9 @@ public C()
                 // (5,19): error CS0548: 'C.P': property or indexer must have at least one accessor
                 //     public string P { }
                 Diagnostic(ErrorCode.ERR_PropertyWithNoAccessors, "P").WithArguments("C.P").WithLocation(5, 19),
+                // (7,19): error CS8050: Only auto-implemented properties can have initializers.
+                //     public string P3 { } = string.Empty;
+                Diagnostic(ErrorCode.ERR_InitializerOnNonAutoProperty, "P3").WithArguments("C.P3").WithLocation(7, 19),
```

Comment:
>Diagnostic(ErrorCode.ERR_InitializerOnNonAutoProperty, "P3").WithArguments("C.P3").WithLocation(7, 19), [](http://example.com/codeflow?start=16&length=103)

This change in behavior doesn't look in line with the goal of the first PR.

Reply:
Fixed